Apps/Encounter/src/Models/Claim.cs

```diff
@@ -0,0 +1,65 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Encounter.Models
+{
+    using System;
+
+    /// <summary>
+    /// Represents a row in the MSP Visit Claim.
+    /// </summary>
+    public class Claim
+    {
+        /// <summary>
+        /// Gets or sets the ClaimId.
+        /// </summary>
+        public int ClaimId { get; set; }
```

Comment:
Looking at our laboratory model we are using ```[JsonPropertyName("mediaType")]``` Lets use that for this ones as well. e.g.
```
[JsonPropertyName("claimId")]
     public int ClaimId { get; set; }
```

Reply:
nice one! forgot that this allows to covert to the json naming format.